.github/workflows/fips-old.yml

```diff
@@ -0,0 +1,74 @@
+# Copyright 2022 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the Apache License 2.0 (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+name: Provider old versions compat
+on: [pull_request, push]
+
+permissions:
+  contents: read
+
+jobs:
+  fips-provider-validation:
+    strategy:
+      matrix:
+        module: [
+          {
+            dir: openssl-3.0.0,
+            tgz: openssl-3.0.0.tar.gz,
+            url: "https://www.openssl.org/source/old/3.0/openssl-3.0.0.tar.gz"
+          },
+        ]
+    runs-on: ubuntu-latest
+    steps:
+      - name: create directory
+        run: mkdir ./current
+      - uses: actions/checkout@v2
+        with:
+          path: current
```

Comment:
Doesn't this create the subdirectory `current` if it's not already there, since this is essentially a `git clone`?  In other words, the `mkdir` above should be unnecessary...

Reply:
We do exactly the same thing in other tests.  I'd prefer consistency at this point.